src/getCoinCombination.test.js

```diff
@@ -7,4 +7,58 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('should return an array', () => {
+    const result = getCoinCombination();
+
+    expect(Array.isArray(result)).toBe(true);
+  });
+
+  it('should return result with 4 char in length', () => {
+    const result = getCoinCombination(10);
+
+    expect(result).toHaveLength(4);
+  });
+
+  it('should return [0, 0, 0, 0] if input is zero', () => {
+    const result = getCoinCombination(0);
+
+    expect(result).toEqual([0, 0, 0, 0]);
+  });
+
+  it('should return [1, 0, 0, 0] for 1 cents', () => {
+    const result = getCoinCombination(1);
+
+    expect(result).toEqual([1, 0, 0, 0]);
+  });
+
+  it('should return [0, 1, 0, 0] for 5 cents', () => {
+    const result = getCoinCombination(5);
+
+    expect(result).toEqual([0, 1, 0, 0]);
+  });
+
+  it('should return [0, 0, 1, 0] for 10 cents', () => {
+    const result = getCoinCombination(10);
+
+    expect(result).toEqual([0, 0, 1, 0]);
+  });
+
+  it('should return [0, 0, 0, 1] for 25 cents', () => {
+    const result = getCoinCombination(25);
+
+    expect(result).toEqual([0, 0, 0, 1]);
+  });
+
+  it('should return [2, 1, 1, 0] for 17 cents', () => {
+    const result = getCoinCombination(17);
+
+    expect(result).toEqual([2, 1, 1, 0]);
+  });
+
+  it('should work properly for big amount of cents', () => {
+    const result = getCoinCombination(258);
+
+    expect(result).toEqual([3, 1, 0, 10]);
```

Comment:
The expected result for 258 cents seems incorrect. According to the task description, the correct combination should be `[3, 1, 0, 10]` (3 pennies, 1 nickel, 0 dimes, and 10 quarters). Please verify and correct this expected value.